SHA256-template.md

```diff
@@ -0,0 +1,25 @@
+Currently, Visual Studio Code extensions are not signed or verified on the
+Microsoft Visual Studio Code Marketplace. Salesforce provides the Secure Hash
+Algorithm (SHA) of each extension that we publish. To verify the extensions,
+make sure that their SHA values match the values in the list below.
+
+1. Instead of installing the Visual Code Extension directly from within Visual
+   Studio Code, download the VS Code extension that you want to check by
+   following the instructions at
+   https://code.visualstudio.com/docs/editor/extension-gallery#_common-questions.
+   For example, download,
+   https://salesforce.gallery.vsassets.io/_apis/public/gallery/publisher/salesforce/extension/salesforcedx-vscode-core/57.15.0/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage.
+
+2. From a terminal, run:
+
+shasum -a 256 <location_of_the_downloaded_file>
+
+3. Confirm that the SHA in your output matches the value in this list of SHAs.
+   SHA_TEMPLATE
```

Comment:
Identical to the actual `SHA256.md` except for this template string; lets us use `sed` to easily update the SHA programatically.

Reply:
Can we use a variable name that says something like SHA_PLACEHOLDER. Or better yet, make it actually look like a placeholder or variable with something like: <<SHA_VALUE>>